src/content/docs/it/core-concepts/astro-components.mdx

```diff
@@ -0,0 +1,347 @@
+
+```astro {14}
+---
+// src/components/Wrapper.astro
+import Header from './Header.astro';
+import Logo from './Logo.astro';
+import Footer from './Footer.astro';
+
+const { title } = Astro.props
+---
+<div id="content-wrapper">
+  <Header />
+  <Logo />
+  <h1>{title}</h1>
+  <slot>
+    <p>This is my fallback content, if there is no child passed into slot</p>
+  </slot>
+  <Footer />
+</div>
+```
+
+### Trasferimento di slot
+
+Gli slot possono essere trasferiti ad altri componenti. Ad esempio, quando si creano layout nidificati:
+
+```astro {11,14}
+// src/layouts/BaseLayout.astro
+---
+---
+
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+		<meta name="viewport" content="width=device-width" />
+		<meta name="generator" content={Astro.generator} />
+    <slot name="head"/>
+	</head>
+	<body>
+		<slot />
+	</body>
+</html>
+```
+
+```astro {7, 8}
+// src/layouts/HomeLayout.astro
+---
+import BaseLayout from "./BaseLayout.astro";
+---
+
+<BaseLayout>
+  <slot name="head" slot="head"/>
+  <slot />
+</BaseLayout>
+```
+
+:::note
+Gli slot con nome possono essere trasferiti a un altro componente utilizzando sia gli attributi `name` che `slot` su un tag `<slot />`
+:::
+
+Ora, gli slot predefiniti e "head" passati a "HomeLayout" verranno trasferiti al genitore "BaseLayout"
+
+```astro
+// src/pages/index.astro
+---
+import HomeLayout from "../layouts/HomeLayout.astro";
+---
+
+<HomeLayout>
+	<title slot="head">Astro</title>
+	<h1>Astro</h1>
+</HomeLayout>
+```
+
+## Componenti HTML
+
+Astro supporta l'importazione e l'utilizzo di file `.html` come componenti o l'inserimento di questi file nella sottodirectory `src/pages/` come pagine. Potresti voler utilizzare componenti HTML se stai riutilizzando il codice di un sito esistente creato senza un framework o se vuoi assicurarti che il tuo componente non abbia funzionalità dinamiche.
+
+I componenti HTML devono contenere solo HTML valido e pertanto non dispongono delle funzionalità chiave dei componenti Astro:
+- Non supportano frontmatter, importazioni lato server o espressioni dinamiche.
+- Tutti i tag `<script>` vengono lasciati non raggruppati, trattati come se avessero `is:inline`.
+- Possono solo [fare riferimento alle risorse che si trovano nella cartella `public/`](/it/core-concepts/project-structure/#public).
+
+:::note
+Un [elemento `<slot />`](/it/core-concepts/astro-components/#slot) all'interno di un componente HTML funzionerà come in un componente Astro. Per utilizzare invece l'elemento [HTML Web Component Slot](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot), aggiungi `is:inline` al tuo `<slot >` elemento.
+:::
+
+## Prossimi passi
+
+📚 Scopri come utilizzare i [componenti del framework UI](/it/core-concepts/framework-components/) nel tuo progetto Astro.
```

Comment:
I think here - and elsewhere - `framework components` means `other framework components`, like React, Svelte, etc.
Hence I would not use `componenti del framework` which makes you think to Astro itself.
I would use `componenti di framework UI`.

Reply:
Gonna do a check if there are other places...

```suggestion
📚 Scopri come utilizzare i [componenti di framework UI](/it/core-concepts/framework-components/) nel tuo progetto Astro.
```